lib/player.rb

```diff
@@ -0,0 +1,10 @@
+
+class Player
+  attr_reader:token
+  def initialize(token)
+    @token =token
+  end
+  def token
```

Comment:
With `attr_reader :token`, this method is not needed. `attr_reader` is a macro that automatically adds both the getter method to your class for the selected attribute. In this case, both
```
def cells
  @token
end
```
has been included in your class through the `attr_reader`.